src/Sylius/Bundle/ApiBundle/Command/PaymentIdAwareInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command;
+
+interface PaymentIdAwareInterface extends CommandAwareDataTransformerInterface
```

Comment:
This interface should not be needed anymore
```suggestion
interface PaymentIdAwareInterface
```